libraries/AP_UAVCAN/AP_UAVCAN.h

```diff
@@ -205,6 +205,7 @@ class AP_UAVCAN : public AP_CANDriver, public AP_ESC_Telem_Backend {
         CANFD_ENABLED             = (1U<<2),
         DNA_IGNORE_UNHEALTHY_NODE = (1U<<3),
         USE_ACTUATOR_PWM          = (1U<<4),
+        SEND_GNSS                 = (1U<<5),
```

Comment:
Need to add this option in the CAN_Dx_US_OPTIONS metadata

Reply:
It already was. What am I missing?